Joedmin-hw3/Joedmin-hw3/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace Joedmin_hw3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                Console.Write("What do you want to do? login/register/exit: ");
+                string answer = Console.ReadLine();
+
+                // Exit
+                if (String.Equals(answer, "exit", StringComparison.OrdinalIgnoreCase))
```

Comment:
`answer.Equals("exit", StringComparison.OrdinalIgnoreCase)` is a little bit more concise.